app/views/swapsea_mailer/offer_received.html.erb

```diff
@@ -129,14 +129,21 @@
           Status: <%= @request.status.capitalize %>
         </p>
         <p>
+          <% if @offer.roster.present? %>
```

Comment:
I think this is one line too high, as it inadvertently removes the "Current Offers" heading.